account_analytic_sequence/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import models
```

Comment:
I though that the checks should fail if there is no new line? May be you can add one

Reply:
One line between license and import is enough, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
ok